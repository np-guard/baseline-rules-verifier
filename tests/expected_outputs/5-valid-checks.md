:white_check_mark:Rule **allow-load-generation** is satisfied

:white_check_mark:Rule **no-ftp** is satisfied

:white_check_mark:Rule **no-telnet** is satisfied

:white_check_mark:Rule **no-smtp** is satisfied

:white_check_mark:Rule **no-imap** is satisfied

All rules are satisfied
