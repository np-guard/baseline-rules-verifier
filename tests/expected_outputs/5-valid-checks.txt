Rule allow-load-generation is satisfied

Rule no-ftp is satisfied

Rule no-telnet is satisfied

Rule no-smtp is satisfied

Rule no-imap is satisfied

All rules are satisfied
